subprojects/docs/src/snippets/plugins/buildscript/kotlin/build.gradle.kts

```diff
@@ -4,9 +4,10 @@ buildscript {
         gradlePluginPortal()
     }
     dependencies {
-        classpath("com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.5")
```

Comment:
:x: I think we should leave the example using maven coordinates directly, maybe showing it last.